admin/api_events.go

```diff
@@ -475,13 +475,13 @@ func (a *EventsApiService) ListEventTypes(ctx context.Context) ListEventTypesApi
 
 // Execute executes the request
 //
-//	@return []EventTypeDetails
-func (a *EventsApiService) ListEventTypesExecute(r ListEventTypesApiRequest) ([]EventTypeDetails, *http.Response, error) {
+//	@return PaginatedEventTypeDetailsResponse
+func (a *EventsApiService) ListEventTypesExecute(r ListEventTypesApiRequest) (*PaginatedEventTypeDetailsResponse, *http.Response, error) {
 	var (
 		localVarHTTPMethod  = http.MethodGet
 		localVarPostBody    interface{}
 		formFiles           []formFile
-		localVarReturnValue []EventTypeDetails
+		localVarReturnValue *PaginatedEventTypeDetailsResponse
```

Comment:
Most impactful change in the PR. Events list is not going to be complete and will require paggination.

Operator, Terraform will need to mitigate paggination when whole list of events is required.
Atlas CLI output will be affected (thus affect CLI based automation)

Reply:
this was always paginated but badly described on openapi, this is also a new API, just released on the last cycle so no-one is using it

Reply:
Released SDK with the same assumption. Thank you for confirmation.